components/video-player.tsx

```diff
@@ -39,164 +9,23 @@ type Props = {
-        hls.loadSource(src);
-        hls.attachMedia(video);
-        hls.on(Hls.Events.ERROR, function (event, data) {
-          if (data.fatal) {
-            logger.error('hls.js fatal error');
-            videoError(new ErrorEvent('HLS.js fatal error'));
-          }
-        });
-      } else {
-        console.error( // eslint-disable-line no-console
-          'This is an old browser that does not support MSE https://developer.mozilla.org/en-US/docs/Web/API/Media_Source_Extensions_API',
-        );
-      }
-
-      if (typeof mux !== 'undefined' && process.env.NEXT_PUBLIC_MUX_ENV_KEY) {
-        mux.monitor(video, {
-          hlsjs: hls,
-          Hls,
-          data: {
-            env_key: process.env.NEXT_PUBLIC_MUX_ENV_KEY,
-
-            player_name: 'hls.js player v1',
-            video_id: playbackId,
-            video_title: playbackId,
-            video_stream_type: 'on-demand',
-            player_init_time: playerInitTime,
-          },
-        });
-      }
-    }
-
-    return () => {
-      if (video) {
-        video.removeEventListener('error', videoError);
-      }
-      if (hls) {
-        hls.destroy();
-      }
-    };
-  }, [playbackId, videoRef]);
-
-  useEffect(() => {
-    const video = videoRef.current;
-    if (currentTime && video) {
-      video.currentTime = currentTime;
-    }
-  }, [currentTime]);
-
   return (
-    <>
-      <div className='video-container'>
-        <video ref={metaRef} poster={poster} controls playsInline />
-      </div>
-      <style jsx>{`
-        :global(:root) {
-          --plyr-color-main: #1b1b1b;
-          --plyr-range-fill-background: #ccc;
-        }
-        :global(.plyr__controls button),
-        :global(.plyr__controls input) {
-          cursor: pointer;
-        }
-        .video-container {
-          margin-bottom: 40px;
-          margin-top: 40px;
-          border-radius: 30px;
-        }
-        :global(.plyr:fullscreen video) {
-          max-width: initial;
-          max-height: initial;
-          width: 100%;
-          height: 100%;
-        }
-        video {
-          display: block;
-          max-width: 100%;
-          max-height: 50vh;
-          cursor: pointer;
-        }
-        @media only screen and (min-width: ${breakpoints.md}px) {
-          video {
-            width: ${isVertical ? 'auto' : '1000px'};
-            height: ${isVertical ? '600px' : 'auto'};
-            max-height: 70vh;
-            min-width: 30rem;
-          }
-        }
-        @media only screen and (max-width: ${breakpoints.md}px) {
-          :global(.plyr__volume, .plyr__menu, .plyr--pip-supported [data-plyr=pip]) {
-            display: none;
-          }
-          video: {
-            width: 100%;
-            height: 100%;
-          }
-        }
-      `}
-      </style>
-    </>
+    <div className='video-container'>
+      <MuxPlayer
+        playbackId={playbackId}
+        poster={poster}
+        muted
+        controls
+        playsInline
```

Comment:
do you know if this is necessary?
> A Boolean attribute indicating that the video is to be played "inline", that is within the element's playback area. Note that the absence of this attribute does not imply that the video will always be played in fullscreen.
  - From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-playsinline